test/formats/test_id3.py

```diff
@@ -74,6 +74,13 @@ def test_id3_multiple_freeform_delete(self):
             self.assertNotIn('Bar', new_metadata)
             self.assertIn('FooBar', new_metadata)
 
+        @skipUnlessTestfile
+        def test_id3_metadata_tofn(self):
+            metadata = Metadata(self.tags)
+            metadata = save_and_load_metadata(self.filename, metadata)
+
+            self.assertIn('originalfilename', metadata)
```

Comment:
Please also check for the expected value (`Foo`) in addition to the presence of the tag